### PR TITLE
Cleanup constructor code in visual shader nodes

### PR DESCRIPTION
**Pull request by Chaosus**
Pushes variable initialization code from various constructors to the class definitions (as we are transferred to C++17).

### CODE REVIEW

File: scene/resources/visual_shader.h [hunk 1]
```diff
@@ -479,10 +479,10 @@ class VisualShaderNodeGroupBase : public VisualShaderNode {
 	void _apply_port_changes();
 
 protected:
-	Vector2 size;
-	String inputs;
-	String outputs;
-	bool editable;
+	Vector2 size = Size2(0, 0);
+	String inputs = "";
+	String outputs = "";
```
Comment (akien-mga) [inline scene/resources/visual_shader.h:484]: `Vector2` is default initialized to `Vector2(0, 0)`, and similarly `String` defaults to `""`, so you don't need to make this explicit.

Comment (Chaosus) [reply scene/resources/visual_shader.h:484]: Yeah, I just remember some cases when the compiler ignores it and pushes uninitialized non-zero data instead.

File: scene/resources/visual_shader_nodes.h [hunk 2]
```diff
@@ -1718,8 +1718,8 @@ class VisualShaderNodeTransformUniform : public VisualShaderNodeUniform {
 	GDCLASS(VisualShaderNodeTransformUniform, VisualShaderNodeUniform);
 
 private:
-	bool default_value_enabled;
-	Transform default_value;
+	bool default_value_enabled = false;
+	Transform default_value = Transform(1.0, 0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0);
```
Comment (akien-mga) [inline scene/resources/visual_shader_nodes.h:1722]: You forgot to remove those from the constructor for `TransformUniform`.

Comment (Chaosus) [reply scene/resources/visual_shader_nodes.h:1722]: Fixed

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**